cypress/support/step_definitions/testSetup.js

```diff
@@ -93,13 +93,14 @@ function destroyAppInstance(testType) {
       params,
       additionalParams
     );
+    cy.log('Sending lifecycle close intent to unload app: ' + JSON.stringify(intentMessage));
 
     try {
       cy.sendMessagetoApp(requestTopic, responseTopic, intentMessage).then((response) => {
```

Comment:
Full intent message is not required here. Just the method and params 
**_Sending lifecycle close intent to unload app, method: lifecycle.close params: { reason: error }_**